### ISSUE TITLE
Add `tabindex=“0”` to slideshow `<ul>`s

### ISSUE THREAD

**Issue by skoolbus39** [unlcms/project-herbie#1128]
Mark [reports](https://unlwdn.slack.com/archives/C05CTFRJ944/p1760116828504479) that he is getting an error in WebAudit that "Scrollable region must have keyboard access." We need to add `tabindex` to slideshow unordered lists in case JavaScript (which programmatically adds tabindex) fails to load or user has JavaScript disabled.

See example [here](https://github.com/unl/wdntemplates/pull/1861).
